tests/pageobjects/side-nav.page.js

```diff
@@ -39,6 +39,10 @@ class SideNav extends Page {
 	get avatar() { return browser.element('[href="/changeavatar"]'); }
 	get preferencesClose() { return browser.element('.sidebar-flex__close-button[data-action="close"]'); }
 
+	get accountStatusBullet() { return browser.element('.sidebar__account-status-bullet'); }
+	get accountStatusBulletOnline() { return browser.element('.sidebar__account-status-bullet--online'); }
+	get accountStatusBulletAway() { return browser.element('.sidebar__account-status-bullet--away'); }
```

Comment:
Why did you add this?

Reply:
Hi, this was meant to be used in e2e tests, but I actually saw that there already were some tests for this functionality. I should have not commited this, I'm removing it.